alacritty/src/config/mod.rs

```diff
@@ -318,6 +319,32 @@ pub fn imports(
     Ok(import_paths)
 }
 
+/// Prune the nulls from the YAML.
```

Comment:
Maybe we should add a comment *why* one would want to do this?

So something like "to ensure toml compatibility" or something.